larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -189,12 +200,13 @@ struct TestParam {
 
 class BaseBConv2DOpModel : public SingleOpModel {
  public:
-  BaseBConv2DOpModel(register_function registration, const TensorData& input,
-                     const TensorData& filter, const TensorData& output,
-                     int stride_width = 1, int stride_height = 1,
-                     enum Padding padding = Padding_VALID,
-                     int dilation_width_factor = 1,
-                     int dilation_height_factor = 1, int num_threads = -1) {
+  BaseBConv2DOpModel(
+      register_function registration, const TensorData& input,
+      const TensorData& filter, const TensorData& output, int stride_width = 1,
+      int stride_height = 1, enum Padding padding = Padding_VALID,
+      enum ActivationFunctionType activation = ActivationFunctionType_NONE,
```

Comment:
why do you need to change the entire function list fo arguments. you can just add it new argument at the end of the list.

Reply:
I wanted to keep it synchronized with the order of the original conv, and therefore i added it after the padding argument. I can move it to the end if you want.